content/en/Getting started/glossary.md

```diff
@@ -152,14 +152,15 @@ penetration testing programs.
 
 ## Pentest Report
 
```

Comment:
I've changed this definition to reflect what's shown in our UI:
<img width="308" alt="Screen Shot 2022-03-09 at 8 43 12 AM" src="https://user-images.githubusercontent.com/84352037/157491581-bba17ada-4e80-4d82-b6bb-d147d1a206fe.png">

Reply:
I have related WIP in [this PR](https://github.com/cobalthq/cobalt-product-public-docs/pull/104/files#diff-7c37a2a0893fd5afb4db9777a0a9996e3c0ce43c80cac2d7c223273683e1eff2)